cabby/data/extract_wikigeo_contexts.py

```diff
@@ -27,7 +27,7 @@
 FLAGS = flags.FLAGS
 FLAGS = flags.FLAGS
 flags.DEFINE_enum(
-    "region", None, ['Pittsburgh', 'Manhattan'],
+    "region", None, ['Pittsburgh', 'Manhattan', 'Bologna'],
     "Map areas: Manhattan or Pittsburgh.")
```

Comment:
Add Bologna.

Reply:
But it was already added. Not sure I understand.

Reply:
Ok. changed